server/pulp/server/webservices/views/decorators.py

```diff
@@ -1,11 +1,3 @@
-"""
-This module contains decorators for web.py class methods.
```

Comment:
This module docstring should be kept (PEP 257) but should be updated for Django. It could even be as simple as:

```
"""
This module contains decorators for Pulp views.
"""
```